CONTRIBUTING.md

```diff
@@ -8,8 +8,9 @@ You will need a "local" version of OpenEMR to make changes to the source code. T
 
 1. [Create your own fork of OpenEMR](https://github.com/openemr/openemr/fork) (you will need a GitHub account) and `git clone` it to your local machine.
     - It's best to also add an `upstream` origin to keep your local fork up to date. [Check out this guide](https://oneemptymind.wordpress.com/2018/07/11/keeping-a-fork-up-to-date/) for more info.
+	- If you haven't already, [install git](https://git-scm.com/downloads) for your system
 2. `cd openemr` (the directory you cloned the code into)
-    - If you haven't already, [install Docker](https://docs.docker.com/install/) for your system
+    - If you haven't already, install [Docker](https://docs.docker.com/install/) and compose(https://docs.docker.com/compose/install/) for your system
 3. Run `docker-compose up` from your command line
```

Comment:
good pts,  could you revert the syntax for 
`install [Docker](https://docs.docker.com/install/)`  so it matches the prior 
`[install Docker](https://docs.docker.com/install/)` and looks like compose could also be changed to a link like [markdown](https://www.markdownguide.org/basic-syntax#links) suggests please? thank you @reidliu41,